app/components/common/paperItem/mobileVenue.scss

```diff
@@ -49,10 +49,14 @@
 }
 
 .sciLabel {
-  font-size: 14px;
+  font-size: 13px;
   font-weight: 500;
   color: $gray800;
 
+  &:before {
```

Comment:
when there aren't both `year` and `Impact Factor`(it's very rare, but...) it makes middle dot.